index.js

```diff
@@ -1,45 +1,48 @@
-/* eslint-disable no-console */
+require('dotenv').config();
+
+if (process.env.USE_CLUSTER === 'true') process.env.USE_CLUSTER = true;
+else process.env.USE_CLUSTER = false;
+
 const { Node } = require('veza');
 const Cluster = require('./src');
-const config = require('./config');
-
-const node = new Node(process.env.CLUSTER)
-    .on('error', (error, client) => console.error(`[IPC] Error from ${client.name}:`, error))
-    .on('socket.disconnect', (client) => console.error(`[IPC] Disconnected from ${client.name}`))
-    .on('socket.destroy', (client) => console.error(`[IPC] Client Destroyed: ${client.name}`))
-    .on('socket.ready', async (client) => {
-        console.log(`[IPC] Connected to: ${client.name}`);
-    });
 
-node.connectTo(config.nodePort).catch((error) => console.error('[IPC] Disconnected!', error));
+if (/^\s*(true|1|on|yes)\s*$/.test(process.env.USE_CLUSTER)) {
```

Comment:
What's the point of this if we are setting the variable to a boolean on L3-L4?

Reply:
env doesn't support booleans, so setting it doesn't do anything. I'll remove at some point.